src/style.css

```
@@ -0,0 +1,181 @@
+
+@font-face {
+  font-family: Avenir;
+  src: url(/fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(/fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+.body_style {
+  margin-top: 20px;
+}
+
+.big_label {
+  display: flex;
+  align-items: left;
+  position: relative;
+}
+
+.bigInput {
+
+  box-sizing: border-box;
+  height: 70px;
+  width: 100%;
+  margin-bottom: 20px;
+  outline: none;
+
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  bottom: 0%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding-left: 62px;
+
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  font-style: normal;
+  color: #3d4e61;
+}
+
+.bigInput:hover {
+
+  box-sizing: border-box;
+  height: 70px;
+  width: 100%;
+  outline: none;
+
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  bottom: 0%;
+
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.bigInput:focus {
+
+  box-sizing: border-box;
+  height: 70px;
+  width: 100%;
+  outline: none;
+
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  bottom: 0%;
+
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: bold;
+}
+
+.bigInput::placeholder {
+
+  left: 0%;
+  right: 18.35%;
+  top: 0%;
+  bottom: -15.79%;
+
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+}
+
+.svg_Big {
```

```suggestion
.svgBig {
```
Use camelCase everywhere. Keep consistency.